src/components/_list-tile.scss

```
@@ -0,0 +1,68 @@
+/*doc
+---
+title: ListTile
+name: list-tile
+category: Components
+---
+
+リストの子要素を表現するコンポーネントです。
+ユーザーアイコンなどの要素を配置できる`leading`と、テキスト用の`title`、他のボタン等の要素を配置できる`trailing`があります。
+
+```example
+<div class="ncgr-list-tile">
+  <div class="ncgr-list-tile__leading">
+    <div class="ncgr-avatar ncgr-avatar--medium">
+      <img class="ncgr-avatar__image" src="./images/components/avatar/avatar.png">
+    </div>
+  </div>
+  <div class="ncgr-list-tile__title">
+    <span>
+      忍者スリスリ君
+    </span>
+    <span class="ncgr-typography-light ncgr-typography-small">
+      surisurikun
+    </span>
+  </div>
+  <div class="ncgr-list-tile__trailing">
+    <i class="fas fa-ellipsis-v" />
+  </div>
+</div>
+```
+
+*/
+
+.ncgr-list-tile {
+  box-sizing: border-box;
+  padding: 8px 16px;
+  width: 100%;
+  height: 64px;
+  display: flex;
+  align-items: center;
+  &:hover {
+    background: var(--color-segmented-control-background);
+  }
+}
+
+.ncgr-list-tile__leading {
+  width: 48px;
+  height: 48px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 16px 0 0;
+}
+
+.ncgr-list-tile__title {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  flex-grow: 1;
+}
+
+.ncgr-list-tile__trailing {
+  width: 48px;
+  height: 48px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

```suggestion
  align-items: center;
```

startの方がきれいに揃ってよさそうかも。